components/ray/ray.go

```diff
@@ -68,6 +73,35 @@ func (r *Ray) GetComponentName() string {
 	return ComponentName
 }
 
+func (r *Ray) GetUpstreamReleaseStatus() status.UpstreamReleases {
+	var componentVersion semver.Version
+	var repositoryURL string
+	var releaseDetails = make([]status.ComponentReleaseStatus, 0)
+	var releaseStatus status.UpstreamReleases
+
+	env, err := godotenv.Read(filepath.Join(deploy.DefaultManifestPath, ComponentName, ".env"))
+
+	if err != nil {
+		return status.UpstreamReleases{}
+	}
+	if env != nil {
+		componentVersion, err = semver.Parse(env["RHOAI_RELEASE_VERSION"])
```

Comment:
Why can't this variable names be the same across all the components? Only ray and dashboard special for some reason